website/docs/r/external_network_v2.html.markdown

```diff
@@ -169,7 +188,11 @@ The following arguments are supported:
 
 * `name` - (Required) A unique name for the network
 * `description` - (Optional) Network friendly description
-* `ip_scope` - (Required) One or more IP scopes for the network. See [IP Scope](#ipscope) below for details.
+* `use_ip_spaces` - (Optional; *v3.10+*; *VCD 10.4.1+*) Defines if the network uses IP Spaces. Do
+* `dedicated_org_id` - (Optional; *v3.10+*; *VCD 10.4.1+*) An Org ID that this network should be
+  not specify `ip_scope` when using IP Spaces. (default `false`)
+* dedicated to. Only applicable when `use_ip_spaces=true`
```

Comment:
There's a strange mix there. I think it is
```suggestion
* `use_ip_spaces` - (Optional; *v3.10+*; *VCD 10.4.1+*) Defines if the network uses IP Spaces. Do not specify `ip_scope` when using IP Spaces. (default `false`)
* `dedicated_org_id` - (Optional; *v3.10+*; *VCD 10.4.1+*) An Org ID that this network should be dedicated to. Only applicable when `use_ip_spaces=true`
```

Reply:
Yes. I moved single line accidentally it looks. Fixed.